packages/thirdweb/src/gas/fee-data.ts

```diff
@@ -113,20 +120,27 @@ export async function getGasOverridesForTransaction(
 export async function getDefaultGasOverrides(
   client: ThirdwebClient,
   chain: Chain,
+  feeType?: FeeType,
 ) {
-  // if chain is in the force gas price list, always use gas price
-  if (!FORCE_GAS_PRICE_CHAIN_IDS.includes(chain.id)) {
-    const feeData = await getDynamicFeeData(client, chain);
-    if (
-      feeData.maxFeePerGas !== null &&
-      feeData.maxPriorityFeePerGas !== null
-    ) {
-      return {
-        maxFeePerGas: feeData.maxFeePerGas,
-        maxPriorityFeePerGas: feeData.maxPriorityFeePerGas,
-      };
-    }
+  // give priority to the transaction fee type over the chain fee type
+  const resolvedFeeType = feeType ?? chain.feeType;
+  // if chain is configured to force legacy transactions or is in the legacy chain list
+  if (
+    resolvedFeeType === "legacy" ||
+    FORCE_GAS_PRICE_CHAIN_IDS.includes(chain.id)
```

Comment:
while you're at it can you add the chain in question to the FORCE_GAS_PRICE_CHAIN_IDS list please?